### ISSUE TITLE
Dark image 

### ISSUE THREAD

**Issue by otalchriis** [pibooth/pibooth#344]
I have a problem!

The images captured by the camera are coming out dark, strange is that in the preview when the timer shows the image is good, only later, both in the original photos folder and in the final file the photo is very dark.

I am using a Logitech C920 webcam.

**Comment by fab1o978** [comment]
Did you try to change ISO values for preview/capture? Not sure they have any effect with a webcam though

**Comment by otalchriis** [comment]
I tried to change the ISO value, it had no effect. I also tried to install some drivers but also notI have another webcam but it is much inferior, strange is that this problem does not happen to her.
Enviado do Yahoo Mail no Android 
 
  Em seg., 9 9e mai. 9e 2022 às 7:10, ***@***.***> escreveu:   


Did you try to change ISO values for preview/capture? Not sure they have any effect with a webcam though
